src/Aspect/ContractCheckerAspect.php

```diff
@@ -32,6 +32,11 @@ class ContractCheckerAspect implements Aspect
     private $reader = null;
 
     /**
+     * @var MethodInvocation
+     */
+    private $invocation;
```

Comment:
Could you please describe, why do you need to store an invocation object in the private property?